cmd/evm/main.go

```diff
@@ -163,12 +163,24 @@ var stateTransitionCommand = &cli.Command{
 				t8ntool.InputAllocFlag,
 			},
 		},
-		&cli.Command{
+		{
 			Name:    "verkle-key",
 			Aliases: []string{"V"},
 			Usage:   "compute the verkle tree key given an address and optional slot number",
 			Action:  t8ntool.VerkleKey,
 		},
+		{
+			Name:    "verkle-code-chunk-key",
```

Comment:
Agree to moving this to another `verkle` command with sub-commands.
Reg `tree` vs `tree-keys`, I'd prefer the latter since `verkle tree` isn't really explaining much what is doing.